tests/target/legacy-control-style-wrap.rs

```diff
@@ -0,0 +1,12 @@
+// rustfmt-max_width: 100
+// rustfmt-control_style: Legacy
+// Ensure legacy control style produces correct wrapping
+
+fn main() {
+    let aaaaaaaaaa_bbbbbb_cccccc_ddddd = if aaaaaaa_bbbb.cc_dddd() &&
```

Comment:
Thanks for you reply. Hmm, TBH the non-default style is not formally specified and I cannot be certain about which is expected here. I think it over that we should follow the style in Configurations.md, so this need not be fixed.